### PR TITLE
Update zmon-aws-agent

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Changes: 
 * include quota name in the servicequota entity

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
👍

File: cluster/manifests/zmon-aws-agent/deployment.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: "visibility"
   labels:
     application: "zmon-aws-agent"
-    version: "v77-12-g3bf9a28-zv156"
+    version: "v77-14-g9daa9d1-zv156
```
Comment (mikkeloscar) [inline cluster/manifests/zmon-aws-agent/deployment.yaml]: string looks wrong with missing `"` in the end?

Comment (aermakov-zalando) [reply cluster/manifests/zmon-aws-agent/deployment.yaml]: Oops :( sorry

Comment (aermakov-zalando) [reply cluster/manifests/zmon-aws-agent/deployment.yaml]: Fixed!

**Comment by linki** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**